libs/utils/federated.js

```diff
@@ -28,3 +28,17 @@ export const getFederatedContentRoot = () => {
 
   return federatedContentRoot;
 };
+
+// TODO we should match the akamai patterns /locale/federal/ at the start of the url
+// and make the check more strict.
+export const getFederatedUrl = (url = '') => {
```

Comment:
It would be great to start a unit test file for `federated.js` covering those functions. They should already be covered within the `utilities.test.js` within the navigation context, but given that we extract them to be consumed anywhere - it makes sense to have it's own test file

Reply:
Thank you, @bandana147 !